src/D2L.Security.OAuth2/Keys/Default/EcDsaPrivateKeyProvider.cs

```diff
@@ -2,11 +2,16 @@
 using System.IdentityModel.Tokens;
 using System.Security.Cryptography;
 using System.Threading.Tasks;
+using D2L.CodeStyle.Annotations;
 
 namespace D2L.Security.OAuth2.Keys.Default {
 	internal sealed partial class EcDsaPrivateKeyProvider : IPrivateKeyProvider {
 
 		private readonly ID2LSecurityTokenFactory m_d2lSecurityTokenFactory;
+		[Mutability.Audited(
+			"Todd Lang",
+			"14-Aug-2018",
+			".Net class we can't mark.")]

```

Comment:
If this is a .Net class that is immutable, why did it not join all the other .Net classes in the exemptions list in the analyzer code?
https://github.com/Brightspace/D2L.CodeStyle/blob/master/src/D2L.CodeStyle.Analyzers/Immutability/KnownImmutableTypes.cs